tests/ui/asm/arm-low-dreg.rs

```diff
@@ -1,28 +1,17 @@
+//@ add-core-stubs
 //@ build-pass
 //@ compile-flags: --target=armv7-unknown-linux-gnueabihf
 //@ needs-llvm-components: arm
-#![feature(no_core, rustc_attrs, decl_macro, lang_items)]
+#![feature(no_core)]
 #![crate_type = "rlib"]
-#![no_std]
```

Comment:
no_std is redundant because no_core implies no_std.

(Although it is mentioned in example in rustc-dev-guide: https://github.com/rust-lang/rustc-dev-guide/blob/master/src/tests/minicore.md)